src/protocol.cpp

```diff
@@ -108,7 +107,6 @@ static const char* ppszTypeName[] =
     NetMsgType::QCOMPLAINT,
     NetMsgType::QJUSTIFICATION,
     NetMsgType::QPCOMMITMENT,
-    NetMsgType::QDEBUGSTATUS,
```

Comment:
Should keep unused inv types here,
s/ `NetMsgType::QDEBUGSTATUS` / `"qdebugstatus", // was only shortly used on testnet` /

Reply:
Fixed